src/app/config/statefile.c

```diff
@@ -132,6 +132,13 @@ static config_var_t state_vars_[] = {
   VAR("CircuitBuildTimeBin",          LINELIST_S, BuildtimeHistogram, NULL),
   VAR("BuildtimeHistogram",           LINELIST_V, BuildtimeHistogram, NULL),
 
+  V(IPv4AutoFail,                     UINT,     "0"),
```

Comment:
The state file variables are unsigned int, but the struct is int. Use uint32_t for both.